lib/src/soloud.dart

```diff
@@ -1333,6 +1350,57 @@ interface class SoLoud {
     _controller.soLoudFFI.setFftSmoothing(smooth);
   }
 
+  /////////////////////////////////////////
+  /// voice groups
+  /////////////////////////////////////////
+
+  /// Used to create a new voice group. Returns 0 if not successful.
+  SoundHandle createVoiceGroup() {
+    final ret = _controller.soLoudFFI.createVoiceGroup();
+    return ret;
+  }
+
+  /// Deallocates the voice group. Does not stop the voices attached to the
+  /// voice group.
+  ///
+  /// [handle] the group handle to destroy.
+  void destroyVoiceGroup(SoundHandle handle) {
+    return _controller.soLoudFFI.destroyVoiceGroup(handle);
+  }
+
+  /// Adds voice handle to the voice group. The voice handles can still be
+  /// used separate from the group.
+  /// [voiceGroupHandle] the group handle to add the new [voiceHandles].
+  /// [voiceHandles] voice handle to add to the [voiceGroupHandle].
+  void addVoicesToGroup(
+    SoundHandle voiceGroupHandle,
+    List<SoundHandle> voiceHandles,
+  ) {
+    return _controller.soLoudFFI.addVoicesToGroup(
+      voiceGroupHandle,
+      voiceHandles,
+    );
+  }
+
+  /// Checks if the handle is a valid voice group. Does not care if the
+  /// voice group is empty.
+  ///
+  /// [handle] the group handle to check.
+  /// Return true if [handle] is a group handle.
+  bool isVoiceGroup(SoundHandle handle) {
+    return _controller.soLoudFFI.isVoiceGroup(handle);
+  }
+
+  /// Checks whether a voice group is empty. SoLoud automatically trims
+  /// the voice groups of voices that have ended, so the group may be
```

Comment:
Just to be clear: does the trimming happen also in SoLoud C++? I'd prefer we stay as close to the underlying library as possible in this respect.

Reply:
Yes, I took this doc from the SoLoud C++ html page.